components/brave_new_tab_ui/helpers/newTabUtils.ts

```diff
@@ -97,6 +97,10 @@ export const generateGridSitesFromLegacyEntries = (
   }
 
   for (const topSite of legacyTopSites) {
+    if (topSite == null) {
```

Comment:
TypeScript is complaining, could you take a look? https://travis-ci.org/github/brave/brave-core/jobs/673060881#L260